CoursesManager.UI/Factory/ViewModelFactory.cs

```diff
@@ -40,12 +54,31 @@ public ViewModelFactory(
             _addressRepository = addressRepository;
             _messageBroker = messageBroker;
             _dialogService = dialogService;
+            _configurationService = configurationService;
+
         }
 
         public T CreateViewModel<T>(object? parameter = null) where T : class
         {
             return typeof(T) switch
             {
+                Type vmType when vmType == typeof(ConfigurationViewModel) =>
+                    new ConfigurationViewModel(
+                        _configurationService,
+                        _dbServer,
+                        _dbPort,
+                        _dbUser,
+                        _dbPassword,
+                        _dbName,
+                        _mailServer,
+                        _mailPort,
+                        _mailUser,
+                        _mailPassword,
+                        _appConfig
```

Comment:
Als ik zo even in de ConfigurationViewModel kijk kun je dit weglaten. Is toch altijd null namelijk.

Reply:
Done